pom.xml

```diff
@@ -138,6 +138,14 @@
       </execution>
     </executions>
   </plugin>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-compiler-plugin</artifactId>
+      <configuration>
+        <source>11</source>
+        <target>11</target>
+      </configuration>
```

Comment:
I think this relates to the java version? I'd recommend to define a java property then and employ it here